CONTRIBUTING.md

```diff
@@ -78,19 +87,66 @@ Enhancement suggestions are tracked as GitHub issues. Make sure you provide the
 
 ## Your First Code Contribution
 
-Unsure where to begin contributing to Booster? You can start by looking through these beginner and help-wanted issues:
+Unsure where to begin contributing to Booster? You can start by looking through issued tagged as `beginner` and `help-wanted`:
```

Comment:
```suggestion
Unsure where to begin contributing to Booster? You can start by looking through issued tagged as `beggood-first-issue` and `help-wanted`:
```

The good first issue is a universal "meta-label" that is used throughout hackathons in GitHub, like Hacktoberfest